### ISSUE TITLE
[REQUEST]: Multiple broadcast/stream management

### ISSUE THREAD

**Issue by Bluesoul02** [Unity-Technologies/UnityRenderStreaming#918]
### Is your feature request related to a problem?

I would like to have a receiver with multiple broadcast and be able to switch the current broadcast received with another, i don't know if this is currently possible by editing and creating a script similar to single connection? This may be related to #641

**Comment by karasusan** [comment]
Yes, this issue is related to #641. Thanks for sharing your issue.

**Comment by karasusan** [comment]
memo: URS-323

**Comment by JohnnyNgo** [comment]
@Bluesoul02  Not sure if related, but out of the box I wasn't able to change a User's streamed screen to another streamed screen during **RUNTIME** (my project is set up with multiple VideoStreamSenders, and when a User connects I choose which stream they should see using AddSender method from SignalingHandlerBase). This worked only when a user connects, but when playing, attempting to RemoveSender then AddSender would just freeze the game. 

To solve this specific issue, I had to implement my solution in #930 , and then update AddSenderCoroutine code
![image](https://github.com/Unity-Technologies/UnityRenderStreaming/assets/43874041/a1cd7b9f-b20f-4671-8df9-68bf067ba350)
and create/use this handly ReplaceSenderTrack method
![image](https://github.com/Unity-Technologies/UnityRenderStreaming/assets/43874041/48da11a9-f11c-4971-9a77-a5a9396eda4e)
![image](https://github.com/Unity-Technologies/UnityRenderStreaming/assets/43874041/0fe0ae36-ec63-4796-98a1-594f5d8e902b)

**Comment by Bluesoul02** [comment]
@JohnnyNgo Thanks! I had the same issue, this is exactly what i needed. I'll try it.
